.travis.yml

```diff
@@ -8,6 +8,7 @@ go_import_path: github.com/Colstuwjx/awx-go
 
 install:
   - go get -u github.com/alecthomas/gometalinter
+  - go get -u github.com/kylelemons/godebug/pretty
```

Comment:
Should we add this dependency inside travis, rather than adding into vendor?
It may caused `go test -v *.go` such way broken.